src/views/chats/ClosedChats/RoomsTable.vue

```diff
@@ -37,7 +37,7 @@
           <template #date>{{ $d(new Date(item.ended_at)) }}</template>
 
           <template #visualize>
-            <div v-if="isMobile">
+            <div v-if="isMobile" @click="$emit('open-room', item)" @keypress.enter="() => {}">
```

Comment:
the `@keypress.enter` Is it supposed to be empty?

Reply:
You're right, it should be the same functionality as `@click`